src/components/Booking/EventTypeGroup/index.js

```diff
@@ -1,23 +1,33 @@
 import React, { Component } from 'react';
 import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { PropTypes as PT } from 'prop-types';
 import { Icon } from 'react-native-elements';
 import { GREY, WHITE } from '../../../styles/colors';
+import type from '../../../constants/eventTypes';
 
 class EventTypeGroup extends Component {
+  static propTypes = {
+    selectEventType: PT.func.isRequired,
+  }
+
   constructor() {
     super();
     this.state = {
       eventType: [
-        { type: 'Annual leave', icon: 'suitcase', color: '#A7BF35' },
-        { type: 'Working from Home', icon: 'home', color: '#399BB6' },
+        { type: type.ANNUAL_LEAVE, icon: 'suitcase', color: '#A7BF35' },
+        { type: type.WFH, icon: 'home', color: '#399BB6' },
         { type: 'Sick leave', icon: 'bed', color: '#A2798F' },
```

Comment:
worth adding these 2 other types to the constants while you are in the code?

Reply:
yes, i will make that change